app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ChargesController < ApplicationController
+  def new
+    @project = Project.find(params[:project_id])
+  end
+
+  def create
+    @project = Project.find(params[:project_id])
+
+    customer = Stripe::Customer.create(
+      :email => params[:stripeEmail],
+      :source  => params[:stripeToken]
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Operator `=>` should be surrounded by a single space.